rasa/core/events/__init__.py

```diff
@@ -1167,6 +1166,23 @@ def apply_to(self, tracker: "DialogueStateTracker") -> None:
         tracker.change_form_to(self.name)
 
 
+class Form(ActiveLoop):
+    """Legacy handler of old `Form` events.
+
+    The `ActiveLoop` event used to be called `Form`. This class is there to handle old
+    legacy events which were stored with the old type name `form`.
+    """
+
+    type_name = "form"
+
+    def as_dict(self) -> Dict[Text, Any]:
+        d = super().as_dict()
+        # Dump old `Form` events as `ActiveLoop` events instead of keeping the old
+        # event type.
+        d["event"] = ActiveLoop.type_name
+        return d
```

Comment:
should we have a deprecation warning for this event though?

Reply:
I wouldn't because users can't do much about this without doing a database migrations. If there are old `Form` events in the database there would be a warning whenever we'd interact with them (e.g. loading the tracker or return it via endpoint).

Reply:
ok fair